setup.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from setuptools import setup
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+with open('HISTORY.rst') as history_file:
+    history = history_file.read()
+
+requirements = [
+    'Click>=6.0',
+    # TODO: put package requirements here
+]
+
+test_requirements = [
+    # TODO: put package test requirements here
+]
+
+setup(
+    name='ricecooker',
+    version='0.1.0',
```

Comment:
I like the pattern where you import the package directly and just uses the version from there. So something like:

```
import ricecooker

setup(
    ...
    version=ricecooker.__version__
```